libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2023,16 +2023,23 @@ void TypeChecker::endVisit(Literal const& _literal)
 {
 	if (_literal.looksLikeAddress())
 	{
-		if (_literal.passesAddressChecksum())
+		if (_literal.value().length() != 42) // "0x" + 40 hex digits
+		{
+			m_errorReporter.syntaxError(_literal.location(),
+										"This looks like an address but is not exactly 20 bytes long. Current length is " +
```

Comment:
Please indent exactly one tab and add a newline right after the `(`. Also single-statement blocks should not have `{`, `}`.